xija/core_1.c

```diff
@@ -76,6 +79,21 @@ int calc_model_1(int n_times, int n_preds, int n_tmals, double dt,
                             mvals[i3][j] = 0.0;
                         }
                         break;
+                    case 5:  /* Node to node radiative coupling */
+                        if (i2 < n_preds && i1 < n_preds) {
+                            tempk1 = y[i1] + c2k;
```

Comment:
This is where you could make this unitless and of order 1 and get rid of the `sb_const`.
```suggestion
                            tempk1 = (y[i1] + c2k) / 300;
```

Reply:
Done, Thanks!